traveltimepy/dto/common.py

```diff
@@ -0,0 +1,122 @@
+from datetime import datetime, time
+from enum import Enum
+from typing import NewType, List, Union, Optional
+from typing_extensions import Literal
+from pydantic.main import BaseModel
+
+SearchId = NewType('SearchId', str)
```

Comment:
I don't see the need for those new, specific types. Python is driven more by conventions than strict typing, isn't it? Does compiler check it? It's cumbersome to import LocationId and initiating it every time. Since we're trying to align the 3.0 sdk with what Python programmers expect, do we want this?

Reply:
They are not strict. It's used more like helper to distinguish types